client/packages/coldchain/src/Equipment/DetailView/AppBarButtons.tsx

```diff
@@ -32,14 +32,14 @@ export const AppBarButtonsComponent = () => {
   };
 
   const printAssetLabel = () => {
+    const date = new Date().toLocaleString().split(', ')[0];
```

Comment:
```suggestion
    const date = new Date().toLocaleDateString();
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString

I think this is a safer thing to do, I'm not sure if all languages will work with the split function...